lib/metanorma/standoc/requirement.rb

```diff
@@ -0,0 +1,65 @@
+module Metanorma
+  module Standoc
+    class Requirement
+      def satisfied
+        raise NotImplementedError("abstract method")
+      end
+
+      def cmd
+        raise NotImplementedError("abstract method")
+      end
+    end
+
+    class LatexmlRequirement < Requirement
```

Comment:
Can we move `LatexmlRequirement` to a separate file? Thanks!

Reply:
Done